Sources/LanguageServerProtocol/Requests/TextDocumentContentRequest.swift

```diff
@@ -10,22 +10,22 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// Request from the client to the server asking for contents of a URI having a custom scheme **(LSP Extension)**
+/// Request from the client to the server asking for contents of a URI having a custom scheme
 /// For example: "sourcekit-lsp:"
 ///
 /// - Parameters:
 ///   - uri: The `DocumentUri` of the custom scheme url for which content is required
 ///
-/// - Returns: `GetReferenceDocumentResponse` which contains the `content` to be displayed.
+/// - Returns: `TextDocumentContentResponse` which contains the `content` to be displayed.
 ///
 /// ### LSP Extension
 ///
 /// This request is an extension to LSP supported by SourceKit-LSP.
-/// Enable the experimental client capability `"workspace/getReferenceDocument"` so that the server responds with
+/// Enable the experimental client capability `"workspace/textDocumentContent"` so that the server responds with
 /// reference document URLs for certain requests or commands whenever possible.
```

Comment:
I don't think that the LSP Extension Heading is no longer necessary here. Am I missing something?

Reply:
No, you're right. Thanks!